src/components/CalendarBar.tsx

```diff
@@ -50,7 +50,7 @@ const CalendarBar: FunctionComponent<CalendarUIProps> = ({
             dispatch({ type: CalendarAction.TogglePicker });
           }}
         >
-          <Typography component="h6">
+          <Typography variant="h6">
```

Comment:
Don't include this in this feature branch.

Reply:
removed